elasticsearch/es_util.py

```diff
@@ -7,10 +8,19 @@
 # to configure Elasticsearch.
 
 
-def prepare_cluster(config):
+def prepare_cluster(config, runtime):
 
-  all_in_one_url = "https://download.elastic.co/downloads/eck/1.1.2/all-in-one.yaml"
-  k8_util.kubectl_run_command(config, f"apply -f {all_in_one_url}")
+  # Prep the eck-operator image
+  docker_util.prepare_image("docker.elastic.co/eck/eck-operator:1.1.2", config)
+  docker_util.format_all_in_one_yaml("docker.elastic.co/eck/eck-operator:1.1.2", config)
+  
+  # Prep the elasticsearch image
+  runtime["elasticsearch_image"] = docker_util.get_gcr_image_path(
```

Comment:
Do you have a notion of why this would be useful to write to the runtime file?
Off-hand it feels like an internal detail.

Reply:
I agree with you that its not particular useful to be in the file. But, in format_cluster_yaml we need to know the gcr.io path of the elasticsearch image, so I thought it'd be useful for it to exist in the runtime just for this. 

Alternatively, we could have get_gcr_image_path live in gen_util, but it seems more of a docker_util thing.